pkg/tenant/db/members.go

```diff
@@ -52,7 +53,52 @@ func (m *Member) UnmarshalValue(data []byte) error {
 	return msgpack.Unmarshal(data, m)
 }
 
-// CreateMember adds a new Member to the database.
+func (m *Member) Validate() error {
+	if m.TenantID.Compare(ulid.ULID{}) == 0 {
+		return ErrMissingTenantID
+	}
+
+	memberName := m.Name
+
+	if memberName == "" {
+		return ErrMissingMemberName
+	}
```

Comment:
Nice! Let's add a test case for this and to the other models so that we protect ourselves against regressions when we make future changes to the validation code.